client/src/pages/Friends.js

```diff
@@ -1,27 +1,99 @@
-import React from "react";
-import { Box, Typography } from "@material-ui/core";
+import React, { useState } from "react";
+import { Box, Typography, Tabs,Tab } from "@material-ui/core";
 import { makeStyles } from "@material-ui/core/styles";
 
+import Followers from '../components/Followers'
+import Following from "components/Following";
+import rose from '../img/roseBarron.png';
+import david from '../img/davidRichardson.png'
+import christine from '../img/christineSmith.png'
+import jennifer from '../img/jenniferFordham.png'
+import rebbeca from  '../img/rebbecaSchindler.png'
```

Comment:
If you change these to absolute imports so:

`'../img/roseBarron.png'`

becomes:

`'img/roseBarron.png'`

it allows the files to be more modular so they aren't dependent on their own position.